.gitmodules

```diff
@@ -84,3 +84,9 @@
 [submodule "external/xxHash"]
 	path = external/xxHash
 	url = https://github.com/Cyan4973/xxHash
+[submodule "external/dynarmic"]
+	path = external/dynarmic
+url=https://github.com/Vita3K/dynarmic.git
```

Comment:
The line format looks wrong to me.
Also, why a fork? Is it really necessary?

Reply:
Fork is very likely to be necessary. Not sure about formatting. Should I just manually format this?

Reply:
i have see and already fixed it in poroper branch here
https://github.com/Zangetsu38/Vita3K/commits/dynarmic
i can force push in your branch if yoiu are right :)

Reply:
Manual formatting is fine.
I understand a fork is the quicker way to add your changes but wouldn't it better to add them upstream and have feedbacks from dynarmic's maintainers?

Reply:
We'll definitely pr to upstream. But, feedbacks can be little slow when Mary is busy. I think having a fork can help us experiment with changes in vita3k early on.